test/unit/runner_tests.rb

```diff
@@ -232,6 +232,35 @@ class InitTests < UnitTests
       assert_equal exp, subject.headers['Content-Type']
     end
 
+    should "set a cookie header with `set_cookie`" do
+      name, value = Factory.string, Factory.string
+      opts        = { Factory.string => Factory.string }
+      exp_headers = {}
+      Rack::Utils.set_cookie_header!(
+        exp_headers,
```

Comment:
@kellyredding - Did you decide not to add the test helper that does this?

Reply:
@jcredding like we discussed, the test helper would be Assert-specific and we don't typically add those here.  I plan on adding it to our internal Deas customizations gem.